oap-server/server-bootstrap/src/main/resources/fetcher-prom-rules/self.yaml

```diff
@@ -79,3 +79,23 @@ metricsRules:
     exp: persistence_timer_bulk_execute_latency_count.sum(['service', 'instance']).increase('PT5M')
   - name: instance_persistence_prepare_count
     exp: persistence_timer_bulk_prepare_latency_count.sum(['service', 'instance']).increase('PT5M')
+  - name: instance_jvm_thread_live_count
+    exp: jvm_threads_current.sum(['service', 'instance'])
+  - name: instance_jvm_thread_daemon_count
+    exp: jvm_threads_daemon.sum(['service', 'instance'])
+  - name: instance_jvm_thread_peak_count
+    exp: jvm_threads_peak.sum(['service', 'instance'])
+  - name: instance_jvm_thread_runnable_state_thread_count
+    exp: jvm_threads_state.tagMatch('state', 'RUNNABLE').sum(['service', 'instance'])
+  - name: instance_jvm_thread_blocked_state_thread_count
+    exp: jvm_threads_state.tagMatch('state', 'BLOCKED').sum(['service', 'instance'])
+  - name: instance_jvm_thread_waiting_state_thread_count
+    exp: jvm_threads_state.tagMatch('state', 'WAITING').sum(['service', 'instance'])
+  - name: instance_jvm_thread_timed_waiting_state_thread_count
+    exp: jvm_threads_state.tagMatch('state', 'TIMED_WAITING').sum(['service', 'instance'])
+  - name: instance_jvm_class_loaded_class_count
+    exp: jvm_classes_loaded.sum(['service', 'instance'])
+  - name: instance_jvm_class_total_unloaded_class_count
+    exp: jvm_classes_unloaded_total.sum(['service', 'instance'])
+  - name: instance_jvm_class_total_loaded_class_count
+    exp: jvm_classes_loaded_total.sum(['service', 'instance'])
```

Comment:
Consider removing `instance_` as a prefix, as JVM has indicated it is instance level.
For example, change `instance_jvm_thread_waiting_state_thread_count` to `jvm_thread_waiting_count`. Remove unnecessary words. The different storages have different rules.